{"name": "test_search_users", "status": "broken", "statusDetails": {"message": "TypeError: APIClient.get() got an unexpected keyword argument 'params'", "trace": "api_context = <playwright._impl._fetch.APIRequestContext object at 0x00000198A0C26C00>\n\n    @pytest.mark.api\n    def test_search_users(api_context):\n        users_api = UserSearchAPI(api_context)\n        query = \"testuser\"\n>       response = users_api.search_users(query=query)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\ntest_users_api.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <api.users_api.UserSearchAPI object at 0x00000198A4247350>, query = 'testuser'\n\n    def search_users(self, query):\n        params = {\"query\": query}\n>       return self.get(self.base_endpoint, params=params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       TypeError: APIClient.get() got an unexpected keyword argument 'params'\n\n..\\api\\users_api.py:18: TypeError"}, "start": 1759125415805, "stop": 1759125415805, "uuid": "f2bb122d-a936-4e8a-9725-cd7df1b9aa23", "historyId": "1a317c8f6b6b80032330356347f323b3", "testCaseId": "1a317c8f6b6b80032330356347f323b3", "fullName": "tests.test_users_api#test_search_users", "labels": [{"name": "tag", "value": "api"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_users_api"}, {"name": "host", "value": "PC-2993"}, {"name": "thread", "value": "2200-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_users_api"}], "titlePath": ["tests", "test_users_api.py"]}